src/org.hdfgroup.object/hdf/object/Datatype.java

```diff
@@ -950,6 +954,15 @@ public boolean isBitField() {
         return (datatypeClass == Datatype.CLASS_BITFIELD);
     }
 
+    /*
+     * (non-Javadoc)
+     *
+     * @see hdf.object.DataFormat#clear()
+     */
+    @Override
+    public void clear() {
+    }
```

Comment:
Not sure, but might be worth throwing an exception here like we do in other base classes where clear isn't implemented.

Reply:
Makes sense, unless it shouldn't be there in the first place.